CMakeLists.txt

```diff
@@ -853,9 +858,6 @@ if(NOT MSVC)
     string(APPEND CMAKE_CXX_FLAGS " -Wno-range-loop-analysis")
     string(APPEND CMAKE_CXX_FLAGS " -Wno-pass-failed")
   endif()
-  if(CMAKE_COMPILER_IS_GNUCXX AND NOT (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 7.0.0))
-    string(APPEND CMAKE_CXX_FLAGS " -Wno-stringop-overflow")
```

Comment:
The double negation in the condition means that this one should be always kept no?

Reply:
Thank you, replaced it with `append_cxx_flag_if_supported`